.chloggen/fix-support-ddtags-batching-logs.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: exporter/datadog
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add fix for change to support ddtags when batching logs, and added unit tests.
```

Comment:
Please read this as a user (not a maintainer) of the Datadog Exporter, and rephrase accordingly. Place yourself in the shoes of a person updating the OpenTelemetry Collector Contrib on his infrastructure, wanting to know "what's new?", not a person who gets involved with the code. What was the problem? What was fixed? Under what conditions did the problem arise? etc.

Reply:
After thinking about this more and reading `CONTRIBUTING.md`, I don't think this PR needs a changelog. The change in functionality is minor and only affects efficiency (number of submit log calls), and the rest is adding unit tests. @mx-psi could you add the skip changelog label if you agree?

Reply:
If there are no changes in behavior and the change is small enough it's fine from my side to skip the changelog